### ISSUE TITLE
`Connect port dies if not connected to in time` is failing on iOS

### ISSUE THREAD

**Issue by baydet** [thaliproject/Thali_CordovaPlugin#1340]
This test is failing because this case is not handled on native. This behavior is described for [`connect`](https://github.com/thaliproject/Thali_CordovaPlugin/blob/master/thali/NextGeneration/thaliMobileNative.js#L240) but not for multiConnectResolved. Should we update native code to match these requirements?

**Comment by yaronyg** [comment]
Yes, we need to have some logic that doesn't just keep a MCSession that nobody is actually using open. But since this isn't really going to hurt battery life as we exclusively run in the foreground we can fix this later.
